bindings/python/iota_sdk/client/_high_level_api.py

```diff
@@ -44,20 +47,24 @@ def as_dict(self):
 
 class HighLevelAPI():
 
-    def get_outputs(self, output_ids: List[OutputId]):
+    def get_outputs(self, output_ids: List[OutputId]) -> List[OutputWithMetadata]:
         """Fetch OutputResponse from provided OutputIds (requests are sent in parallel).
         """
-        return self._call_method('getOutputs', {
-            'outputIds': output_ids
+        result =  self._call_method('getOutputs', {
+            'outputIds': list(map(lambda o: o.output_id, output_ids))
         })
+        outputs = [from_dict(OutputWithMetadata, o) for o in result]
+        return outputs
 
-    def get_outputs_ignore_errors(self, output_ids: List[OutputId]):
+    def get_outputs_ignore_errors(self, output_ids: List[OutputId]) -> List[OutputWithMetadata]:
         """Try to get OutputResponse from provided OutputIds.
            Requests are sent in parallel and errors are ignored, can be useful for spent outputs.
         """
-        return self._call_method('getOutputsIgnoreErrors', {
-            'outputIds': output_ids
+        result =  self._call_method('getOutputsIgnoreErrors', {
+            'outputIds': list(map(lambda o: o.output_id, output_ids))
         })
+        outputs = [from_dict(OutputWithMetadata, o) for o in result]
+        return outputs
```

Comment:
rename `result` to `outputs`, and then just
```
return [from_dict(OutputWithMetadata, o) for o in outputs]
``

Reply:
Yes we can!

Reply:
Updated